monkey/monkey_island/cc/models/report.py

```diff
@@ -18,7 +23,7 @@ class Report(Document):
     @staticmethod
```

Comment:
Should we do the same thing with these that we did with `telemetry_dal.py`?

Reply:
The same should also be done with configuration (we should have a DAL for it, but this refactoring would never end)